VoteCounter/src/VoteCount.cs

```diff
@@ -57,12 +63,40 @@ private void findVotes(Post post)
             {
                 if (isVoteValid(match.Groups[1].Value))
                 {
+                    
+                    string foundPlayer = new string('a',1);
```

Comment:
Why is this defaulting to "a"? Especially since the check below checks for an empty string!